src/components/SingleArticle/SingleArticle.tsx

```diff
@@ -30,6 +31,9 @@ export const SingleArticle = ({ article }: { article: ArticleRecord }) => {
 
   return (
     <article className={styles.SingleArticle}>
+      {!isPublished(article) && (
+        <h2 className={styles.previewBanner}>Unpublished Article Preview</h2>
```

Comment:
Not sure about the wording here, let me know if something else would be better

Reply:
works for me!